redis/src/connection.rs

```diff
@@ -599,14 +705,18 @@ impl ActualConnection {
                 ref host,
```

Comment:
same for line 703 - shouldn't it be `#[cfg(any(feature = "tokio-rustls-comp", feature = "tokio-native-tls-comp"))]`?

Reply:
I think in this case `tls-native-tls` is right since this isn't async code. The `tokio-*-comp` feature names aren't used in this file.